packages/localize/src/number/getGroupSeparator.js

```diff
@@ -12,6 +12,6 @@ export function getGroupSeparator(locale) {
   const formattedNumber = Intl.NumberFormat(computedLocale, {
     style: 'decimal',
     minimumFractionDigits: 0,
-  }).format('1000');
-  return normalSpaces(formattedNumber[1]);
+  }).format('10000');
```

Comment:
Why is the extra 0 needed if I may ask?

Reply:
sure you can :) it is on the commit message. The norm says (you can check it on this [wikipedia](https://en.wikipedia.org/wiki/Decimal_separator#Exceptions_to_digit_grouping) link) you can omit the separator group when you have a four digit number, and for some reason in the last update of V8, they have decided to follow this optional rule, at least for es-ES. What always returns the separator is a more than four digit number, so the extra 0 makes always the job.

Reply:
Instead of returning the second position of the formatted four digit number, you can return the third position of a five digit number, and you skip the possible customization. 5 digit numbers must be formatted with the separator.